contrib/kafka/config/README.md

```diff
@@ -3,12 +3,11 @@
 Deployment steps:
 
 1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
-1. If not done already, install an Apache Kafka cluster. There are two choices:
+1. If not done already, install an Apache Kafka cluster!
 
-   - Simple installation of [Apache Kafka](broker).
-   - A production grade installation using the
-     [Strimzi Kafka Operator](http://strimzi.io). Installation
-     [guides](http://strimzi.io/quickstarts/) are provided for kubernetes and
+   - For Kubernetes a simple installation is done using the
+     [Strimzi Kafka Operator](http://strimzi.io). It's installation
+     [guides](http://strimzi.io/quickstarts/) provide content for kubernetes and
      Openshift.
```

Comment:
Add a sentence here that you can also use an off-cluster Kafka, just so it's clear that Kafka on Kubernetes is not the only option.